packages/beacon-node/test/utils/runEl.ts

```diff
@@ -15,7 +15,7 @@ export enum ELStartMode {
   PostMerge = "post-merge",
 }
 
-export type ELSetupConfig = {mode: ELStartMode; elScriptDir: string; elBinaryDir: string};
+export type ELSetupConfig = {mode: ELStartMode; elScriptDir: string; elBinaryDir: string; template?: string};
```

Comment:
template variable name maybe too generic
```suggestion
export type ELSetupConfig = {mode: ELStartMode; elScriptDir: string; elBinaryDir: string; genesisTemplate?: string};
```